project-koku/koku
# [COST-2559] strip acct off ebs account num for hcs csv's per IT request

Fixes [COST-2559](https://issues.redhat.com/browse/COST-2559)

Changes proposed in this PR:
* Strip the `acct` prefix off the EBS Account Number in the CSV's for IT per a request from IT
* 

Testing Instructions:
1. Spin up koku locally with Trino, such as: make docker-up-min-trino

2. After everything spins up, ingest some Azure data:
        - make create-test-customer
        - make test_source=azure load-test-customer-data
3. Check minio to see that you have no HCS data in the koku-bucket (there would be a folder called HCS next to the data one) http://localhost:9090/login

4. Add your account to unleash under the hcs-data-processor http://localhost:4242/#/features/strategies/hcs-data-processor

5. Hit the masu endpoint for hcs and put in your Azure source info: http://localhost:5042/api/cost-management/v1/hcs_report_data/?start_date=2022-04-01&schema=acct10001&provider_type=Azure&provider_uuid=02d80185-78ea-45c5-b75b-925786d26b7d

6. Check minio again and this time there is a path down HCS that leads to some daily Azure csvs based on the date you used int he masu endpoint. The `ebs_account_id` in these csv's should be `10001` instead of `acct10001` 


---
Additional Context:
